archivebox/main.py

```diff
@@ -129,6 +129,14 @@
     FAVICON_FILENAME,
 }
 
+
+def using_django(func):
```

Comment:
this is cool but I think it's too "magic" for not much gain. easier to just keep it as a function call at the top of all the ones that need it.

```python3
def add(urls: Union[str, List[str]], out_dir: Path):
    setup_django(out_dir=our_dir, check_db=True)
    ...
```